app/models/note.rb

```diff
@@ -18,9 +19,23 @@ class Note < ApplicationRecord
   translates :body
   include Translatable
 
+  cattr_accessor :default_style, default: 'original'
+  cattr_accessor :style_labels, default: {
+    'Original': 'original',
+    '🔵 Blue': 'blue',
+    '🔴 Red': 'red',
+    '🟢 Green': 'green',
+    '🟡 Yellow': 'yellow'
+  }
```

Comment:
We discussed this `unstyled` option, instead of using Notes, we're going to look to adding a rich text `Category#body` which can be displayed unstyled after the notes which are being added in #7255.